docs/concepts/economics/index.md

```diff
@@ -45,7 +45,7 @@ _Users_ execute session and contract code using the platform's computational res
 
 ### Incentives (runtime layer) {#incentives-runtime-layer}
 
-The Casper node software can be configured to support various fee, refund, and cost-handling strategies. The Condor release on Mainnet has enabled a [fee elimination](./fee-elimination.md) model by default, setting the `no_fee`,`no_refund`, and `fixed` pricing configurations in the network's chainspec.
+The Casper node software can be configured to support various fee, refund, and cost-handling strategies. Subject to governance voting, a future release(after Casper 2.0 launch) will enable fee elimination model by default, setting the `no_fee`,`no_refund`, and `fixed` pricing configurations in the network's chainspec.
```

Comment:
add space between `release` and `(`

Reply:
replace `(after Casper 2.0 launch) with `(after Casper 2.0 mainnet activation)`

Reply:
Thanks, incorporated it.